### DISCUSSION TITLE
Fcrn knockout mouse

### DISCUSSION THREAD

**Discussion by kajasir** [Open-Systems-Pharmacology/Forum#1416] [Questions & Problems]
Hello,
this is a really basic question but I can't figure out how to create a Fcrn knockout mouse (individual) in PK-Sim. I probably just missed something but I could really use some help.
I created a compound (molecule) that has a very high Kd for Fcrn which is just approximately similar outcome but it is not accurate enough.

Thank you for any information
Kaja

**Comment by Christoph27** [comment]
Hi Kaja,

you can set the "Start concentration of free FcRn" to 0 at the individual. For the location see attached screenshot. 
It is visible in the "Advanced" view, see bottom left corner of the screenshot.


![SetFcRnKO](https://user-images.githubusercontent.com/26439150/209087109-12c90525-015e-49ee-add1-987ad77d302d.jpg)
